build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.85.1'
+    gutenbergMobileVersion = 'v.1.86.0-alpha-1'
```

Comment:
I think there's a typo here as the Git tag for [the alpha version is `v1.86.0-alpha1`](https://github.com/wordpress-mobile/gutenberg-mobile/releases/tag/v1.86.0-alpha1), there's an extra `.` at the beginning of the string.

Reply:
Thanks for pointing this out! There was also a dash between `alpha` and `1`. 😅 I've made some updates, and the CI is now passing. 👍